### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by gidsi**
i've created the mail alias temporarily, it's a forward to the mailboxes of @dbrgn @rnestler @slopjong and @gidsi

### CODE REVIEW

File: CODE_OF_CONDUCT.md
Comment (rnestler) [inline CODE_OF_CONDUCT.md:37]: Should we mention chat.spaceapi.net here as well as a way to contact the project team?

Comment (dbrgn) [reply CODE_OF_CONDUCT.md:37]: For this type of contact I'd stick to e-mail.

**Review by dbrgn — APPROVED** [review]
Good to go from me. I think this needs an explicit approval from every team member before we merge it though.

**Comment by dbrgn** [comment]
By the way, merging this into the codebase is a bit moot if the domain points to the other server now :)

**Review by rnestler — APPROVED** [review]
:+1:

**Comment by rnestler** [comment]
Should we merge this here? Or create a separate repository or create a spaceapi.io repository and add it there?

**Comment by dbrgn** [comment]
I'd add it to a new `website` repo. I'll create it in a few minutes with the source from the spacedirectory website.

**Comment by dbrgn** [comment]
@rnestler @gidsi one of you can now move it to https://github.com/SpaceApi/website/!

**Pull request closed without merging**
